astropy/tests/helper.py

```diff
@@ -465,7 +464,7 @@ def assert_quantity_allclose(actual, desired, rtol=1.e-7, atol=None,
                                                                rtol, atol),
                                **kwargs)
 
-
+@deprecated("3.1.dev21533")
```

Comment:
* Just "3.1" is sufficient.
* Please provide a string to explain where it has moved to.
* To avoid code duplication, this function now should only have one functional line of code to just call the new one.